### PR TITLE
Even more underscores

### PR DESCRIPTION
**Pull request by rustyrussell**
More trivial things I missed with my simple regex search, plus a name consistency update.

### CODE REVIEW

File: 03-transactions.md [hunk 1]
```diff
@@ -86,14 +86,14 @@ This output sends funds back to the owner of this commitment transaction, thus m
         # Penalty transaction
         <revocation-pubkey>
```
Comment (pm47) [inline 03-transactions.md]: Just to be sure, are you keeping dashes in `<revocation-pubkey>`, `<local-delayedkey>`, `<ripemd-of-payment_hash>` on purpose? 

Should we also fix inconsistencies between pubkey and key? We have both `<revocation-key>` and `<revocation-pubkey>`.

Ther are also inconsistencies between `<revocation-sig>` and `<revocationkey-hash>`. If that's outside the scope of this PR I'll submit another one.

File: 03-transactions.md [hunk 2]
```diff
@@ -120,14 +120,14 @@ This output sends funds to a HTLC-timeout transaction after the HTLC timeout, or
             OP_DROP 2 OP_SWAP <localkey> 2 OP_CHECKMULTISIG
         OP_ELSE
             # To you with preimage.
-            OP_HASH160 <ripemd-of-payment-hash> OP_EQUALVERIFY
+            OP_HASH160 <ripemd-of-payment_hash> OP_EQUALVERIFY
```
Comment (Roasbeef) [inline 03-transactions.md]: Dashes here should actually be underscores?

File: 03-transactions.md [hunk 3]
```diff
@@ -148,11 +148,11 @@ This output sends funds to the remote peer after the HTLC timeout or using the r
             OP_SIZE 32 OP_EQUAL
         OP_IF
             # To me via HTLC-success transaction.
-            OP_HASH160 <ripemd-of-payment-hash> OP_EQUALVERIFY
+            OP_HASH160 <ripemd-of-payment_hash> OP_EQUALVERIFY
```
Comment (Roasbeef) [inline 03-transactions.md]: Missed an `underscore -> dashes` substitution here.

File: 03-transactions.md (hunk 1 above)
Comment (rustyrussell) [inline 03-transactions.md]: Hmm, I'll fix those too.  Was just aiming for actual defined field names, but that was kind of silly.

File: 03-transactions.md (hunk 2 above)
Comment (rustyrussell) [inline 03-transactions.md]: Technically, I was trying to replace all field names (payment_hash is a field of update_add_htlc).  But I think we should actually put <RIPEMD160(payment_hash)> here.  Will add a new commit...

**Comment by rustyrussell** [comment]
OK, feedback incorporated, more conversions.  I'm tempted to leave any remaining fixups for a separate typo PR later :)

**Comment by rustyrussell** [comment]
Assigned to @Roasbeef: if you're happy, please apply...

**Pull request merged**